test/elixir_auth_google_test.exs

```diff
@@ -22,14 +22,21 @@ defmodule ElixirAuthGoogleTest do
   end
 
   test "get Google login url" do
-    Application.put_env(:elixir_auth_google, :google_client_id, 123)
```

Comment:
Also removed it from this older test as no longer required. (_that way it won't confuse us in the future ..._) 👍